### DISCUSSION TITLE
Calificacion Actividad Practica S3P2

### DISCUSSION THREAD

**Discussion by gonaiot** [ISPC-TST-EM2022/Grupo2#45] [Announcements]
Calificacion Grupal : 6 

@Jerec4stro : 7  
@WayarCarla : 7
@dr-arriola : 5
@danydeitu : 5
@NataliaGalliani : 5

Observaciones: 
Estimados, como grupo veo que les esta costando mucho organizarse, recuerden que la practica se evalua a nivel personal y a nivel colaborativo. Organizarse no quiere decir realizar una sola persona todas las tareas, Debe haber una persona que genera el proyecto de la actividad, que necesariamete realizara la estructura de carpetas, subcarpetas y archivos. Y dentro del proyecto, se generan las tareas (issues) que los demas integrantes deben ir tomando. Lean bien, el propósito de la actividad; la misma pide que cada estudiante resuelva por lo menos una punto o tarea.  Por lo cual completando las tareas teóricas del ejercicio #2 habrían cumplido, individualmente con lo solicitado. Y varios no realizaron aunque sea esto. 

Si Uds se dirigen al menu insights podran ver puntualmente la participacion colectiva e individual de cada uno, y si se van a pull requests, podran ver como trabajaron con respecto a la practica puntual. Saludos.

**Comment by WayarCarla** [comment]
Buenas profesor en el aula del instituto podrá subir el cronograma de como
se van a realizar las siguientes actividades, más que nada el link para
acceder a las clases sincrónicas y asincronicas,( en esta última sus
horarios ). De esta forma me voy a organizar para ver si así logro realizar
las actividades al nivel que ustedes lo requieren. Ya no estoy en discord
así que me voy a guiar por la plataforma oficial.
Saludos.
Carla
